user/getting-started.md

```diff
@@ -85,6 +80,46 @@ $ echo "import foo" > sample.py
 $ bblfsh client sample.py
 ```
 
+## Babelfish Tools
+
+Babelfish Tols provide some language analysis tools on top of Babelfish. You can
+use them for various purposes:
+
+- Check that the server is working properly
+- Get some data from the source code
+- See how a language analysis tool is implemented, as a basis for your own
+  tools.
+
+
+### Setup
+
+Running Babelfish Tools standalone requires getting the `bblfsh-tools`
+binary. Currently this requires a working setup of
+[Go](https://golang.org/). You can get it with the following command:
```

Comment:
Maybe this link is more direct for setting up Go: https://golang.org/doc/install